service/static/index.html

```diff
@@ -88,7 +88,7 @@ <h3>Create, Retrieve, Update, and Delete an Inventory:</h3>
             <!-- SUBMIT BUTTONS -->
             <div class="form-group">
               <div class="col-sm-offset-2 col-sm-10">
-                <button type="submit" class="btn btn-primary" id="listAll-btn">List All</button>
+                <button type="submit" class="btn btn-primary" id="listall-btn">List All</button>
```

Comment:
This new `id` listall-btn won't match with the name in `rest_api.js` . Either change in both places or leave it as `listAll`

Reply:
https://github.com/CSCI-GA-2820-FA22-001/inventory/blob/bdd_base/service/static/js/rest_api.js#L206
listAll-btn is dependent on this^

Reply:
reverted , doesn't matter in this PR